examples/nodeset/optStructs_test.c

```diff
@@ -0,0 +1,180 @@
+//
+// Created by grigory on 22.01.20.
+//
```

Comment:
Adjust the header to the open62541 specific format. Maybe we should make an more generic example with opt structs which doesn't use the auto id nodeset.